include/inexor/vulkan-renderer/wrapper/window.hpp

```diff
@@ -0,0 +1,86 @@
+#pragma once
+
+#include <GLFW/glfw3.h>
+#include <spdlog/spdlog.h>
+#include <vulkan/vulkan.h>
+
+#include <cassert>
+
+namespace inexor::vulkan_renderer::wrapper {
+
+class Window {
+private:
+    VkInstance instance;
```

Comment:
Does the instance need to be a member (or even part of the class at all). I don't see anywhere where it's used.

Reply:
Right. It's not even used anywhere in the code. That's a leftover from having window+surface in one class.

Reply:
Done.